api/src/main/java/org/openmrs/module/registrationcore/api/mpi/common/MpiPatient.java

```diff
@@ -0,0 +1,20 @@
+package org.openmrs.module.registrationcore.api.mpi.common;
+
+import org.openmrs.Patient;
+
+/**
+ * Defines a MpiPatient in the system. A mpi patient is simply an extension of a patient and all that that
+ * implies.
+ */
+public class MpiPatient extends Patient {
```

Comment:
@djazayeri yep, that would make sense for us, if we need to store more values in MPI patients.
But, the reason why I inherit MpiPatient from Patient is:
When we call "get similar patients" from registrationApp we return a list of patients. In that list are some "local" patients, and some "mpi" patients. Since MpiPatient inherits from Patient we can put it in that list, and by getting "isMpiPatient" we can figure out is Patient MPI or local.